hedera-node/src/main/java/com/hedera/services/state/org/StateMetadata.java

```diff
@@ -51,10 +57,14 @@ public StateMetadata copy() {
 
 	@Override
 	public void release() {
-		// No-op
+		aliases.release();
 	}
 
 	public ServicesApp app() {
 		return app;
 	}
+
+	public FCHashMap<ByteString, EntityNum> aliases() {
```

Comment:
Nit, better for return type to be `Map<ByteString, EntityNum>` unless caller really needs to do some operations that are only possible on an `FCHashMap` type. Same goes for sibling methods in `ServicesState.java` and `StateMetadata.java`, and the constructor in `AliasManager.java`.

Reply:
Ah excellent point, only `StateMetadata` actually needs to know that it's dealing with an `FCHashMap`. Fixed.